tests/console/zypper_extend.pm

```diff
@@ -104,12 +104,13 @@ sub run {
     zypper_call 'renamerepo "packman" Packman';
 
     #Remove a repository
-    zypper_call 'rr packman';
+    zypper_call 'rr Packman';
 
     #Verify whether all dependencies are fulfilled
     zypper_call 'verify';
 
     #Identify processes and services using deleted files
+    zypper_call 'in lsof';
```

Comment:
`zypper` should probably install `lsof` as a dependency and we should install it here only as a workaround and record a `soft_fail`, no?

Reply:
> `zypper` should probably install `lsof` as a dependency and we should install it here only as a workaround and record a `soft_fail`, no?

I am not sure, as I cannot spot lsof as a dependency of zypper (using the --requires option on zypper). Maybe I am wrong, however if indeed lsof is a dependecy of zypper, probably the issue should be reported ^^

Reply:
> > `zypper` should probably install `lsof` as a dependency and we should install it here only as a workaround and record a `soft_fail`, no?
> 
> I am not sure, as I cannot spot lsof as a dependency of zypper (using the --requires option on zypper). Maybe I am wrong, however if indeed lsof is a dependecy of zypper, probably the issue should be reported ^^

Small update :) After discussing with @dzedro , a bug will be opened for Tumbleweed JeOS. Therefore, it will be tracked as a soft_fail, as proposed by @ggardet.

Reply:
> Small update :) After discussing with @dzedro , a bug will be opened for Tumbleweed JeOS. Therefore, it will be tracked as a soft_fail, as proposed by @ggardet.

No need to open a bug, it is already there https://build.opensuse.org/package/view_file/openSUSE:Factory:ToTest/kiwi-templates-JeOS/JeOS.kiwi?expand=1.

`lsof` is missing only in JeOS images based on sle15sp1 and leap15.1
[Proposal](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/a9dc27b1c0011c9ac5f0052867d16b3d0aa5aa3b/tests/console/zypper_extend.pm#L114)

Reply:
Ah, I see, you are dealing with [the community JeOS](https://openqa.opensuse.org/tests/1301261#)